lib/routes/account.js

```diff
@@ -1021,50 +1045,78 @@ module.exports = function (
         validate: {
           payload: {
             uid: isA.string().max(32).regex(HEX_STRING).required(),
-            code: isA.string().min(32).max(32).regex(HEX_STRING).required()
+            code: isA.string().min(32).max(64).regex(HEX_STRING).required()
```

Comment:
> Can the tokens be one and the same? 

@rfk wrote the same thing [down here](https://github.com/mozilla/fxa-auth-server/pull/1232/files#r60523308) along with all the reasons @philbooth and I discussed why verifying a session must verify the account too.